test/test_tensor_creation_ops.py

```diff
@@ -2782,6 +2782,20 @@ def test_logspace(self, device, dtype):
         y = torch.logspace(0, 3, 4, base=2, device=device, dtype=dtype, out=x.narrow(1, 1, 2))
         self.assertEqual(x, torch.tensor(((0, 1, 2), (0, 4, 8)), device=device, dtype=dtype), atol=0, rtol=0)
 
+        # Integer test
+        if torch.testing.is_integral(dtype):
```

Comment:
you can just create a separate test 
```
@skipCPUIf(True, "compares with CPU")
@dtypesIfCUDA(torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64)
def test_logspace_integral(self, device, dtype):
  ...
```

see how other special typed logspace test were doing before this one.

Reply:
I added this. I chose to not skip all CPU tests because there are some parts not concerning CPUs.